src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,3 @@
+@mixin transition-navLink-prodCard {
+  transition: transform $transition-speed, color $transition-speed;
+}
```

Comment:
no needs to create this mixin, since for header hover you don't use `transform $transition-speed`